azext_edge/edge/_help.py

```diff
@@ -857,101 +801,122 @@ def load_iotops_help():
+        long-summary: The file name will be {asset_name}_{dataset_name}_dataPoints.{file_type}.
         examples:
         - name: Export all data points in an asset in JSON format.
           text: >
-            az iot ops asset data-point export --asset {asset} -g {resource_group}
+            az iot ops asset dataset point export --asset myasset -g myresourcegroup --dataset dataset1
         - name: Export all data points in an asset in CSV format in a specific output directory.
           text: >
-            az iot ops asset data-point export --asset {asset} -g {resource_group} --format csv --output-dir {output_directory}
+            az iot ops asset dataset point export --asset myasset -g myresourcegroup --dataset dataset1 --format csv --output-dir myAssetsFiles
         - name: Export all data points in an asset in CSV format that can be uploaded via the DOE portal.
           text: >
-            az iot ops asset data-point export --asset {asset} -g {resource_group} --format portal-csv
+            az iot ops asset dataset point export --asset myasset -g myresourcegroup --dataset dataset1 --format portal-csv
         - name: Export all data points in an asset in YAML format. Replace the file if one is present already.
           text: >
-            az iot ops asset data-point export --asset {asset} -g {resource_group} --format yaml --replace
+            az iot ops asset dataset point export --asset myasset -g myresourcegroup --dataset dataset1 --format yaml --replace
     """
 
     helps[
-        "iot ops asset data-point import"
+        "iot ops asset dataset point import"
     ] = """
         type: command
-        short-summary: Import data points in an asset.
+        short-summary: Import data points in an asset dataset. ### TODO: update once dup param is decided
         long-summary: For examples of file formats, please see aka.ms/aziotops-assets
         examples:
-        - name: Import all data points from a file. These data points will be appended to the asset's current data points. Data-points with duplicate dataSources will be ignored.
+        - name: Import all data points from a file. These data points will be appended to the asset dataset's current data points. Data-points with duplicate dataSources will be ignored.
           text: >
-            az iot ops asset data-point import --asset {asset} -g {resource_group} --input-file {input_file}
-        - name: Import all data points from a file. These data points will be appended to the asset's current data points. Data-points with duplicate dataSources will be replaced.
+            az iot ops asset dataset point import --asset myasset -g myresourcegroup --dataset dataset1 --input-file myasset_dataset1_dataPoints.csv
+        - name: Import all data points from a file. These data points will be appended to the asset dataset's current data points. Data-points with duplicate dataSources will be replaced.
```

Comment:
Is there a need for support of replacing all dataset data points? Not just ones with duplicate data source.

Reply:
actually right now it is a full replace (I think need to double check). 

tbh not sure if we should support full replace vs partial replace vs append - if all then at that point I would need an enum...